src/core/PlayArea.ttslua

```diff
@@ -130,4 +130,5 @@ end
 
 function onScenarioChanged(scenarioName)
   currentScenario = scenarioName
+  Global.call('titleSplash', {scenarioName = scenarioName})
```

Comment:
Let's call this in MythosArea as part of the direct event, rather than deferring to here